python/ray/worker.py

```diff
@@ -1501,6 +1504,11 @@ def connect(node,
         # Remove excludes, it isn't relevant after the upload step.
         runtime_env.pop("excludes", None)
         job_config.set_runtime_env(runtime_env)
+	
+    if _default_actor_lifetime is not None:
+        job_config.set_default_actor_lifetime(_default_actor_lifetime)
```

Comment:
We always overwrite the user-specified default actor lifetime in job config? That doesn't sound good.

Reply:
Addressed.